checker/src/org/checkerframework/checker/units/UnitsAnnotatedTypeFactory.java

```diff
@@ -303,14 +272,11 @@ private boolean isUnitsMultiple(AnnotationMirror metaAnno) {
      */
     private void addUnitsRelations(Class<? extends Annotation> qual) {
         AnnotationMirror am = AnnotationUtils.fromClass(elements, qual);
-        Class<org.checkerframework.checker.units.qual.UnitsRelations> unitsRelationsAnnoClass =
-                org.checkerframework.checker.units.qual.UnitsRelations.class;
 
         for (AnnotationMirror ama : am.getAnnotationType().asElement().getAnnotationMirrors() ) {
             if (ama.getAnnotationType().toString().equals(unitsRelationsAnnoClass.getCanonicalName())) {
```

Comment:
Is there some reason that AnnotationUtils.areSameByClass is insufficient for this check?